pkg/rule/rules_nodejs/grpc_web_js_library.go

```diff
@@ -0,0 +1,63 @@
+package rules_nodejs
+
+import (
+	"github.com/bazelbuild/bazel-gazelle/config"
+	"github.com/bazelbuild/bazel-gazelle/label"
+	"github.com/bazelbuild/bazel-gazelle/resolve"
+	"github.com/bazelbuild/bazel-gazelle/rule"
+
+	"github.com/stackb/rules_proto/pkg/protoc"
+)
+
+const (
+	grpcWebJsLibraryRuleName   = "grpc_web_js_library"
+	grpcWebJsLibraryRuleSuffix = "_grpc_web_js_library"
+)
+
+func init() {
+	protoc.Rules().MustRegisterRule("stackb:rules_proto:grpc_web_js_library", &grpcWebJsLibrary{})
+}
+
+// grpcWebJsLibrary implements LanguageRule for the 'grpc_web_js_library' rule from @rules_proto.
+type grpcWebJsLibrary struct{}
+
+// Name implements part of the LanguageRule interface.
+func (s *grpcWebJsLibrary) Name() string {
+	return grpcWebJsLibraryRuleName
+}
+
+// KindInfo implements part of the LanguageRule interface.
+func (s *grpcWebJsLibrary) KindInfo() rule.KindInfo {
+	return jsLibraryKindInfo
+}
+
+// LoadInfo implements part of the LanguageRule interface.
+func (s *grpcWebJsLibrary) LoadInfo() rule.LoadInfo {
+	return rule.LoadInfo{
+		Name:    "@build_stack_rules_proto//rules/nodejs:grpc_web_js_library.bzl",
```

Comment:
the directory `rules/nodejs` implies that the rule implementation is related to `bazelbuild/rules_nodejs`...  is it?

Reply:
edit: looks like it is: `load("@build_bazel_rules_nodejs//:index.bzl", "js_library")`